docs/conan.md

```diff
@@ -0,0 +1,147 @@
+  --profile:host=<b><i>CI/conan/PROFILE</i></b>
+</pre>
+
+The highlighted parts can be adjusted:
+
+- ***conan-generated***: directory where the generated files will appear. This value is used in CMake presets from VCMI, but you can actually use any directory and override it in your local presets.
+- ***never***: use this value to avoid building any dependency from source. You can also use `missing` to build recipes, that are not present in your local cache, from source.
+- ***CI/conan/PROFILE***: if you want to consume our prebuilt binaries, ***PROFILE*** must be replaced with one of filenames from our [Conan profiles directory](../CI/conan) (determining the right file should be straight-forward). Otherwise, either select one of our profiles or replace ***CI/conan/PROFILE*** with `default` (your default profile).
+
+If you use `--build=never` and this command fails, then it means that you can't use prebuilt binaries out of the box. For example, try using `--build=missing` instead.
+
+VCMI "recipe" also has some options that you can specify. For example, if you don't care about game videos, you can disable FFmpeg dependency by passing `-o with_ffmpeg=False`. If you only want to make release build, you can use `GENERATE_ONLY_BUILT_CONFIG=1` environment variable to skip generating files for other configurations (our CI does this).
+
+_Note_: you can find full reference of this command [in the official documentation](https://docs.conan.io/en/latest/reference/commands/consumer/install.html) or by executing `conan help install`.
+
+### Using prebuilt binaries from ConanCenter
+
+First, check if binaries for [your platform](https://github.com/conan-io/conan-center-index/blob/master/docs/supported_platforms_and_configurations.md) are produced.
+
+You must adjust the above `conan install` command:
+
+1. Replace ***CI/conan/PROFILE*** with `default`.
+2. Additionally pass `-o default_options_of_requirements=True`: this disables all custom options of our `conanfile.py` to match ConanCenter builds.
+
+## Configure project for building
+
+You must pass the generated toolchain file to CMake invocation.
+
+- if using custom CMake presets, just make sure to inherit our `build-with-conan` preset. If you store Conan generated files in a non-default directory, define the path to the generated toolchain in `toolchainFile` field (or `CMAKE_TOOLCHAIN_FILE` cache variable) or include CMake presets file generated by Conan.
+- otherwise, if passing CMake options on the command line, use `--toolchain` option (available in CMake 3.21+) or `CMAKE_TOOLCHAIN_FILE` variable.
+
+## Examples
+
+In these examples only the minimum required amount of options is passed to `cmake` invocation, you can pass additional ones as needed.
+
+### Use our prebuilt binaries to build for macOS x86_64 with Xcode
+
+```
+conan install . \
+  --install-folder=conan-generated \
+  --no-imports \
+  --build=never \
+  --profile:build=default \
+  --profile:host=CI/conan/macos-intel
```

Comment:
WARN: boost/1.79.0: requirement libiconv/1.17 overridden by your conanfile to libiconv/1.17@kambala/apple 
libiconv/1.17@kambala/apple: Not found in local cache, looking in remotes...
libiconv/1.17@kambala/apple: Trying with 'conancenter'...
ERROR: Failed requirement 'libiconv/1.17@kambala/apple' from 'boost/1.79.0'
ERROR: Unable to find 'libiconv/1.17@kambala/apple' in remotes